### PR TITLE
Add prerequisite install include for KIC

### PR DESCRIPTION
**Pull request by mheap**
### Description

Add a new collapsible prerequisites section for KIC guides that uses Helm to install KIC + Gateway.

This ensures that all guides will be using the same Kubernetes service/deployment names, rather than leaving the d

### Testing instructions

Netlify link:  /2.11.x/guides/configuring-health-checks/#setup-active-health-checking

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/_includes/md/kic/prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    Content-Type: application/json; charset=utf-8
+    Connection: keep-alive
+    Content-Length: 48
+    X-Kong-Response-Latency: 0
+    Server: kong/3.0.0
+  
+    {"message":"no Route matched with those values"}
+    ```
+
+    If the above instructions do not work for you, please read our more [detailed deployment guides](/kubernetes-ingress-controller/{{ page.release }}/deployment/overview/).
```
Comment (mheap) [inline app/_includes/md/kic/prerequisites.md:66]:
```suggestion
    If the above instructions do not work for you, please read the more [detailed deployment guide](/kubernetes-ingress-controller/{{ page.release }}/deployment/overview/).
```

**Comment by Rajakavitha1** [comment]
Hi @mheap I committed some changes PR . Please take a look.

**Review by Rajakavitha1 — APPROVED** [review]
LGTM.

**Comment by mheap** [comment]
@Rajakavitha1 Most changes LGTM

I can see that you’ve changed the headers to be all h2. I don’t believe that prerequisites carry as much weight as the actual guide, which is why I set them to h3

In general we could do a better job showing hierarchy using headers in these guides. I’d like to set them back to h3 unless you feel strongly (and please do say if you disagree! I don’t feel super-strongly about this)

**Comment by mheap** [comment]
@Rajakavitha1 Aha! Yes, the outline takes the first heading level on the page as the maximum. We'd need a "Prerequisities" h2 with the h3 nested underneath.

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 Aha! Yes, the outline takes the first heading level on the page as the maximum. We'd need a "Prerequisities" h2 with the h3 nested underneath.

@mheap  now it is ready for merge :)

**Comment by mheap** [comment]
Merging now.

> Also, do you think all the guides that use the installation.md include should now be updated to prerequistes.md

Yes please @Rajakavitha1 - the guides will need re-testing once switched to `prerequisites.md`

**Pull request merged**